docs/building-open-mpi/required-support-libraries.rst

```diff
@@ -0,0 +1,128 @@
+.. _install-required-support-libraries-label:
+
+Required support libraries
+--------------------------
+
+Open MPI uses the following support libraries:
+
+* `Hardware Locality (hwloc) <https://www.open-mpi.org/projects/hwloc/>`_
+* `Libevent <https://libevent.org/>`_
+* `PMIx <https://pmix.org/>`_
+* `PRRTE <https://github.com/openpmix/prrte>`_
+
+Full copies of the source code of these libraries are bundled in the
+Open MPI source code distribution for end-user convenience.
+
+* In some cases, the presence of these bundled versions can lead to
```

Comment:
This reads like just having the source bundled can cause problems, which isn't really the case.  I think the complete description of the problem needs to wait until you introduce the dependency chain.  For now, possibly say that shared dependencies can cause problems in very unusual configurations.

Reply:
I'll just remove these 2 bullets.